features/messages/requests/MarkAllReadButton.tsx

```diff
@@ -73,15 +73,17 @@ export default function MarkAllReadButton({
       {markAll.error && (
         <Snackbar severity="error">{markAll.error.message}</Snackbar>
       )}
-      <Tooltip title={t("mark_all_read_button_text")}>
+
+      <Button variant="text" onClick={() => markAll.mutate()}>
         <IconButton
-          aria-label={t("mark_all_read_button_text")}
+          aria-label={"mark messages as read"}
```

Comment:
why this change?
arial label isn't visible anymore to the user, so I just needed to change the text so the current tests work properly

Reply:
Since you moved the label out (it's not hidden anymore like you said), you can remove this label altogether. Otherwise, screen readers will read this out twice. For non-English users (eventually when we turn translations on), they'll hear both languages side-by-side (since you've now hardcoded the English message here) which will be a very confusing experience

Reply:
Ahh I see the it's because `IconButton` requires an aria-label. You can use the `Button` component we have (instead of importing directly from `@material-ui/core`) and put the checkmark icon together with the text label as direct children, as it's not necessary to have a button nested inside a button (and it's invalid HTML).